packages/manager/modules/manager-pci-common/src/components/deployment-tiles-input/DeploymentTilesInputForApp.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import { OsdsSpinner } from '@ovhcloud/ods-components/react';
+import {
+  DeploymentTilesInput,
+  DeploymentTilesInputProps,
+} from '@/components/deployment-tiles-input/DeploymentTilesInput';
+import { useFeaturedDeploymentModes } from '@/api/hook/useFeaturedDeploymentModes';
+
+export type DeploymentTilesInputForAppProps = Omit<
+  DeploymentTilesInputProps,
+  'deployments'
+>;
+
+export const DeploymentTilesInputForApp = ({
```

Comment:
This one is an abstraction for µapps, but I feel a bit confused with the name.

BTW, we could consider to move it out of components folder or to find a way to export these kinds of components.

Reply:
Yes, I don't know what is the best way for this, maybe with the `Featued` name as in the hook ?